charts/testkube-cloud-api/values.yaml

```diff
@@ -217,6 +217,8 @@ api:
 
   # -- External log server connection configuration
   logServer:
+    # -- Toggle whether to enable external log server connection
+    enabled: false
```

Comment:
should we also update enterprise values?

Reply:
good catch! updated